### PR TITLE
add explanation how to fully erase bash shell history

### PR DESCRIPTION
**Pull request by matkoniecz**
based on http://unix.stackexchange.com/questions/203290/how-do-i-clear-the-terminal-history/203295#203295

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks! LGTM :+1:

**Comment by igorshubovych** [comment]
Too long description. Can it be shortened?

**Comment by matkoniecz** [comment]
removal of 

> (typically combined with `history -c` to completely erase history)

would be the obvious candidate but I am not sure is it an improvement.

**Comment by waldyrious** [comment]
I agree the parenthetical remark should not be removed. Perhaps shortened: "typically" --> "often", "completely erase" --> "clear" (or "purge").

**Comment by igorshubovych** [comment]
Let me turn it this way.

TLDR is about the most common examples.

@matkoniecz do you usually use `-w`?

Personally I am using `history` quite often. Mostly in pipes. A few times I've used `history -c`. And I have never used `history -w`, and frankly speaking I doubt everybody needs it. But if I am wrong, please tell me.

**Comment by matkoniecz** [comment]
> @matkoniecz do you usually use -w?

Yes, and I keep googling for it. But it is likely that I am in minority that keeps typing password at wrong time and then forgetting to use `history -c` before closing terminal.

**Comment by igorshubovych** [comment]
Ok. Then thanks for contribution! :+1:

**Comment by waldyrious** [comment]
Any thoughts about my proposal to shorten the parenthetical note?

**Comment by matkoniecz** [comment]
> Any thoughts about my proposal to shorten the parenthetical note?

see #828
